mysql-test/suite/rocksdb/t/fast_prefix_index_fetch.test

```diff
@@ -0,0 +1,97 @@
+--source include/have_rocksdb.inc
```

Comment:
This testcase seems to be taken from the sibling feature in InnoDB, and that one is affected by https://github.com/facebook/mysql-5.6/issues/568, which IMHO is rather serious for affected charset users. I don't know if RocksDB implementation is affected by the same - please consider taking https://github.com/percona/percona-server/blob/5.6/mysql-test/suite/innodb/include/prefix_index_only_query_check.inc, https://github.com/percona/percona-server/blob/5.6/mysql-test/suite/innodb/t/percona_fast_prefix_index_fetch.test, https://github.com/percona/percona-server/blob/5.6/mysql-test/suite/innodb/r/percona_fast_prefix_index_fetch.result, which are based on the same testcase but test more charsets and prefix lengths.